src/main/webapp/WEB-INF/jsp/roster.jsp

```diff
@@ -121,26 +123,30 @@
                                 <button id="saveAttendanceOnTop" class="hovering-purple-button" type="button" name="saveAttendance" onclick="submitRoster()">
                                     Save Attendance</button>
                             </div>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)